package.json

```diff
@@ -7,8 +7,7 @@
     "build": "node build.js",
     "webpack": "webpack -p",
     "prepublish": "npm run build",
-    "start": "micro -p 3000 server/index.js",
-    "dev": "nodemon --exec 'npm start'"
```

Comment:
Could https://github.com/zeit/micro-dev/ be used as an alternative?

Reply:
Maybe, I’m not too familiar with the micro ecosystem unfortunately. In any case I would recommend to merge this as soon as possible to avoid exposing more people cloning this repo to the vulnerability.